core-bundle/src/Controller/Page/AbstractLayoutPageController.php

```diff
@@ -44,6 +45,9 @@ public function __invoke(Request $request): Response
             throw $this->createNotFoundException();
         }
 
+        // Load contao_default translations (#8690)
+        $this->getContaoAdapter(System::class)->loadLanguageFile('default', $request->getLocale());
```

Comment:
I don't think we should pass the request locale here, it should use the current translation locale (that should already be _known_ to the framework.